RecoHI/HiEvtPlaneAlgos/python/hievtplaneflatproducer_cfi.py

```diff
@@ -1,7 +1,20 @@
 import FWCore.ParameterSet.Config as cms
```

Comment:
Note that cfi file names should usually match the name of the module instances defined in them
(here hievtplaneflatproducer isn't the same as hiEvtPlaneFlat)